rollup.config.js

```diff
@@ -35,18 +35,21 @@ let lessPlugin = less({
 let intro = `window.addEventListener('WebComponentsReady', function () {`;
 let outro = `});`;
 
+let browserOutput = {
+    format: 'umd',
+    intro,
+    outro,
+    sourcemap: false,
+}
+
 export default [
     // src/browser-entry.js --> dist/helix-ui.browser.js (UMD)
     {
         input: 'src/browser-entry.js',
-        name: 'HelixUI',
-        sourcemap: false,
-        intro,
-        outro,
```

Comment:
These are no longer input options. Rollup throws a fit if `sourcemap` is defined as an input option, but it politely warns for `name`, `intro`, and `outro`.